modules/ocf/manifests/packages/fahclient.pp

```diff
@@ -0,0 +1,17 @@
+class ocf::packages::fahclient {
+  # Folding @ Home
+  ocf::repackage {'fahclient':}
```

Comment:
You shouldn't need `ocf::repackage` here as it's not backporting anything (nor is it backporting only on a particular distribution from what I can tell), just a simple `package { 'fahclient':; }` should do just fine.

Reply:
Oh also you might want to include the package for buster too, although there's no desktops on buster currently, but if it's to be used at all in the future.

Reply:
Yep, included it for both buster and stretch. Thanks!